types/react-native/test/index.tsx

```diff
@@ -350,9 +350,9 @@ export class SectionListTest extends React.Component<SectionListProps<string>, {
                             <Text>{section.title}</Text>
                         </View>
                     )}
-                    renderItem={(info: { item: string }) => (
+                    renderItem={(info: { item: string, section: { title: string, data: string[] } }) => (
```

Comment:
Could you define the new method with a type definition? Like so
```typescript
renderItem = ( item: SectionListRenderItem<> ) => {
   ...
}
```

Reply:
Sure